clouddriver-openstack/src/main/resources/asg.yaml

```diff
@@ -27,10 +27,46 @@ parameters:
     description: Load balancer to use for this group
   security_groups:
     type: comma_delimited_list
-    description: security groups to use
+    description: Security groups to use
   subnet_id:
     type: string
     description: Subnet used to allocate a fixed IP for each server
+  autoscaling_type:
```

Comment:
Good question ... The only benefit to keep them here is that they can be easily seen and executed.